app/controllers/notes_controller.rb

```diff
@@ -2,8 +2,8 @@ class NotesController < ApplicationController
   before_action :check_logged_in
   before_action :set_note, only: [:show, :edit, :update, :destroy]
   def index
-    @notes_by_month = Note.page(params[:page]).per(10).grouped_by_month(current_user)
-    @notes = Note.where(user: current_user).page(params[:page]).per(10)
```

Comment:
No in that case it's fine. If your route `/notes` is not authentication protected, it could happen that sb. who is not authenticated reaches the `index` action.